dotCMS/src/main/java/com/dotcms/integritycheckers/HostIntegrityChecker.java

```diff
@@ -75,8 +74,7 @@ public File generateCSVFile(final String outputPath) throws DotDataException, IO
             writer = new CsvWriter(new FileWriter(csvFile, true), '|');
```

Comment:
Double check if the db annotations are in place

Reply:
Checked and added.